### ISSUE TITLE
Mammals tour: "big up" the placenta on tourstop 6

### ISSUE THREAD

**Issue by hyanwong** [OneZoom/tours#3]
We should put a picture or diagram of a placenta on tourstop 6, and big up what an amazing thing it is.

**Comment by lentinj** [comment]
Seems like the sort of thing that there'd be a good short video explainer for.

**Comment by hyanwong** [comment]
True, but I think we can make a first pass with a diagram (and there are already a number of videos on the tour)

**Comment by hyanwong** [comment]
Maybe fixed by #6
